setup.py

```diff
@@ -24,13 +24,15 @@
     keywords="betfund",
     packages=find_packages(),
     install_requires=[
+        "betfund_logger @ git+https://github.com/betfund/betfund-logger.git",
```

Comment:
We might want to keep this as such, if we do not want to expose `betfund-logger` to all of pypi